exdb/tests/integration_tests.py

```diff
@@ -314,6 +325,35 @@ def test_week_ahead(self):
 
 class ExperienceApprovalViewTest(StandardTestCase):
 
+    def post_data(self, message="", approve=False, invalid_description=False, llc_approval=False):
+        """Posts approval/denial data and returns updated experience for comparisons
+        default value is no comment and deny"""
+        e = self.create_experience('pe', start=(now() + timedelta(days=1)), end=(now() + timedelta(days=2)))
+        status = 'approve' if approve else 'deny'
+        description = "" if invalid_description else e.description
+        next_approver = get_user_model().objects.create_user(
```

Comment:
This user should be created in the `setUp()` method - not here. You can reference it here via a `self.*` attribute.